pkg/yurthub/proxy/util/util.go

```diff
@@ -198,17 +198,11 @@ func WithIfPoolScopedResource(handler http.Handler) http.Handler {
 type wrapperResponseWriter struct {
 	http.ResponseWriter
 	http.Flusher
-	http.CloseNotifier
 	http.Hijacker
 	statusCode int
 }
 
 func newWrapperResponseWriter(w http.ResponseWriter) *wrapperResponseWriter {
-	cn, ok := w.(http.CloseNotifier)
```

Comment:
because Yurthub depends on `httputil.ReverseProxy` for proxying requests, and i have checked golang v1.18.10, `http.CloseNotifier` interface will be used for closing request when context done.
https://cs.opensource.google/go/go/+/refs/tags/go1.18.10:src/net/http/httputil/reverseproxy.go;l=221-233

so it's not a good idea to remove `http.CloseNotifier` field from wrapperResponseWriter, and we need to find a workaround.

Reply:
Yes, this is a problem.

Reply:
I have checked that request.Context is recommended to use for replacing http.CloseNotifier,  it's no problem to remove `http.CloseNotifier`. so i will merge this pull request.